var/spack/repos/builtin/packages/netcdf/package.py

```diff
@@ -39,8 +39,13 @@ class Netcdf(Package):
     version('4.4.0', 'cffda0cbd97fdb3a06e9274f7aef438e')
     version('4.3.3', '5fbd0e108a54bd82cb5702a73f56d2ae')
 
-    variant('mpi',  default=True,  description='Enables MPI parallelism')
-    variant('hdf4', default=False, description='Enable HDF4 support')
+    variant('mpi',     default=True,  description='Enables MPI parallelism')
+    variant('hdf4',    default=False, description='Enable HDF4 support')
+    variant('shared',  default=True,  description='Enable shared library')
+    variant('static',  default=True,  description='Enable static library')
```

Comment:
This is the only package that uses the `static` variant, and if I am not wrong we decided a while ago to use just `shared`. Anyhow: what would be the semantic of
```console
spack install netcdf~shared~static
```
that is now allowed?